### ISSUE TITLE
Usage

### ISSUE THREAD

**Issue by ancker010** [smopucilowski/bikegeo#1]
This project looks stale, but it's extremely interesting to me.

Do you mind updating or adding info to the Usage section?

If I can get it working, I'd gladly contribute a bunch of geometry stats.

**Comment by ancker010** [comment]
I actually figured most of it out. But it would still be nice to see updated usage docs.

**Comment by smopucilowski** [comment]
You're right, it is stale --- this is because it's intended use (frame overlays to compare sizes when shopping) was filled by sites such as:

https://99spokes.com
https://geometrygeeks.bike

However, you're right, I can add some (brief) useage documentation if someone wishes to contribute or extend. I'll keep this issue open until I commit some documentation.

**Comment by ancker010** [comment]
I'd never heard of 99spokes before, but I've seen geometrygeeks and use it often.
99spokes does the overlay thing that does mostly make this tool moot, but nevertheless it was still fun to reverse engineer what you were doing any make it work!
